book/modules/module14-exercises.tex

```diff
@@ -82,6 +82,30 @@
 
 			\item What is $\Null(A^{-1})$? Justify your answer.
 		\end{enumerate}
+\prob Anna and Ella are studying relations between the determinant and the volume. In particular, they are studying $\mathcal{S}:\mathbb{R}^3\rightarrow\mathbb{R}^3$, where $\mathcal{S}\matc{x \\ y \\ z}=\matc{4x \\ 2z \\ 0}$, and $\mathcal{T}:\mathbb{R}^3\rightarrow\mathbb{R}^2$, where $\mathcal{T}\matc{x \\ y \\ z}=\matc{2x \\ 8z}$.
```

Comment:
relations -> the relationship

, where -> defined by

Move the question propmpt up here instead of reapeating it for each part. Rephrase: "For each convesation below, (a) evaluate Anna and Ella's arguments as *correct*, *mostly correct*, or *incorrect*; (b) point out where each argument makes correct/incorrect statements; (c) give a correct numerical value for the determinant or explain why it doesn't exist."